src/UI/UIUtils.java

```diff
@@ -82,10 +82,10 @@ private static void initCosmetics()
 
       // Throw some color names in there for the defaults
       // toString() is not user-friendly
-      paletteNames.put(Color.PINK, "salmon");
+      paletteNames.put(Color.PINK, "rose");
       paletteNames.put(Color.CYAN, "cyan");
-      paletteNames.put(Color.ORANGE, "citrus");
-      paletteNames.put(PURPLE, "sparking");
+      paletteNames.put(Color.ORANGE, "orange");
```

Comment:
Oh, goody
![image](https://user-images.githubusercontent.com/8674241/59555826-8f0a9f00-8f7e-11e9-8556-d1350aec1c2d.png)

Reply:
You just love these mirror matchups, don't you?

Reply:
I dunno, it's just really weird to me.